cherab/core/tests/test_beam.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2016-2023 Euratom
+# Copyright 2016-2023 United Kingdom Atomic Energy Authority
+# Copyright 2016-2023 Centro de Investigaciones Energéticas, Medioambientales y Tecnológicas
+#
+# Licensed under the EUPL, Version 1.1 or – as soon they will be approved by the
+# European Commission - subsequent versions of the EUPL (the "Licence");
+# You may not use this work except in compliance with the Licence.
+# You may obtain a copy of the Licence at:
+#
+# https://joinup.ec.europa.eu/software/page/eupl5
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the Licence is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the Licence for the specific language governing permissions and limitations
+# under the Licence.
+
+import unittest
+
+import numpy as np
+
+from raysect.core import World, Vector3D, translate
+
+from cherab.core import Beam
+from cherab.core.atomic import AtomicData, BeamStoppingRate
+from cherab.core.atomic import deuterium
+from cherab.tools.plasmas.slab import build_constant_slab_plasma
+from cherab.core.model import SingleRayAttenuator
+
+from cherab.core.utility import EvAmuToMS, EvToJ
+
+
+class ConstantBeamStoppingRate(BeamStoppingRate):
+    """
+    Constant beam CX PEC for test purpose.
+    """
+
+    def __init__(self, donor_metastable, value):
+        self.donor_metastable = donor_metastable
+        self.value = value
+
+    def evaluate(self, energy, density, temperature):
+
+        return self.value
+
+
+class TestAtomicData(AtomicData):
```

Comment:
Rename this `MockAtomicData`: classes starting with `Test` are assumed by both unittest and pytest to contain test cases and this convention is broken here which in turn throws up warnings in the test discovery process.